### PR TITLE
[FIX] Jobbaned can't spam-spawn as cult harvesters

### PR DESCRIPTION
**Pull request by larentoun**
## What Does This PR Do

Jobbanned can spam smoke with Nar'Sie, when it should spawn only on harverster spawn. It fixes this.

## Why It's Good For The Game

Jobbanned shouldn't be able to make server lag

## Testing

Nope, just hoping for the best...

## Changelog
:cl:
fix: Jobbanned can't spam NarSie harvester smoke anymore
/:cl:

### CODE REVIEW

File: code/game/gamemodes/wizard/soulstone.dm [hunk 1]
```diff
@@ -425,6 +425,7 @@
 	if(jobban_isbanned(target, ROLE_CULTIST))
 		return
 	var/mob/living/simple_animal/hostile/construct/C = new c_type(get_turf(target))
+	new /obj/effect/particle_effect/smoke/sleeping(target.loc)
```
Comment (SteelSlayer) [inline code/game/gamemodes/wizard/soulstone.dm:428]: I don't think this should happen by default. Maybe attach an arg `creates_smoke = FALSE`. Then when clicking on Narsie, set that to TRUE: `make_new_construct(..., creates_smoke = TRUE)`

Comment (larentoun) [reply code/game/gamemodes/wizard/soulstone.dm:428]: Well, I could do that, if it's needed

Comment (S34NW) [reply code/game/gamemodes/wizard/soulstone.dm:428]: Please do so @larentoun

**Comment by larentoun** [comment]
Oh right, forgot about it, a moment

**Review by lewcc — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**